pkg/atlantis/README.md

```diff
@@ -0,0 +1,146 @@
+# atlantis integration
+
+## Dynamic Repo Config Generation
+
+If you want to generate `atlantis.yaml` before Atlantis can parse it,
+you can add the following run command to [pre_workflow_hooks](https://www.runatlantis.io/docs/pre-workflow-hooks.html#pre-workflow-hooks).
+The `atlantis.yaml` repo config file will be generated right before Atlantis will parse it.
+
+```yaml
+repos:
+  - id: /.*/
+    pre_workflow_hooks:
+      - run: ./repo-config-generator.sh
+```
+
+In `repo-config-generator.sh`, you need to first generate the varfiles for all components in all stacks,
+then generate `atlantis.yaml` repo config file using the following `atmos` commands:
```

Comment:
The `atmos` command would also need to be installed which means the user would need to cut an image of atlantis to install atmos OR atmos can be installed on-the-fly by the `pre_workflow_hook`.

The former method would be nice to have documented as the latter method would be inefficient and would incur a lot more network traffic.

Reply:
I'd like to do both. I'd want to do 1 so the files in git will be able to trigger atlantis and I'd like to do 2 to double check if 1 was done correctly so atmos is using the most up-to-date tfvars when running terraform.